lib/legato/management/finder.rb

```diff
@@ -6,9 +6,22 @@ def base_uri
       end
 
       def all(user, path=default_path)
-        query = Legato::Management::Query.new(base_uri + path, self)
+        items = []
+
+        url = base_uri + path
+
+        loop do
```

Comment:
Could this be a `while (!url.nil?) do` instead of a loop?

Reply:
Or empty …

Reply:
Could this use `map` instead of `each`, and maybe a different name for the collection than `items` … it feels a little weird to match the name from `response.items`. `results` or something is fine …